test/emulate/all.spec.ts

```diff
@@ -170,11 +168,13 @@ describe('Transpose a tar stream', () => {
 			'./test/emulate/test-files/valid-archive.tar',
 		);
 
-		return transpose.transposeTarStream(tarStream, opts).then((stream) => {
-			return expect(getDockerfileFromTarStream(stream)).eventually.to.equal(
-				expectedOutput,
-			);
-		});
+		return transpose
+			.transposeTarStream(tarStream, opts)
+			.then(async (stream) => {
+				return expect(await getDockerfileFromTarStream(stream)).to.equal(
```

Comment:
I probably no longer need this
```suggestion
				expect(await getDockerfileFromTarStream(stream)).to.equal(
```